torso-bundle.js

```diff
@@ -488,7 +488,7 @@
           }
           return $.ajax({
               type: collection.fetchHttpAction,
-              url: collection.url + collection.getByIdsUrl,
+              url: _.result(collection, 'url') + collection.getByIdsUrl,
```

Comment:
It is in the collection mixin. The bundle was created just before pushing.
The person who made the change to the mixin didn't bundle before comitting